### DISCUSSION TITLE
Budibase cli 2.11.33 prebuilds folder and cli version response missing

### DISCUSSION THREAD

**Discussion by gmentzik** [Budibase/budibase#12071] [Help]
Since I have updated Budibase to latest version i wanted also to get the latest cli tool. 
I am using directly the linux version as ode installation is not working for me, but linux package version was absolutely fine.

So i am using : 
 wget https://github.com/Budibase/budibase/releases/latest/download/cli-linux, which currently fetches (version 2.11.33)

I see that when I execute budi script it does not generate the prebuilds folder and also budi -V returns 0.0.0

The previous version I had was v2.4.31 and was generating the prebuilds folder as well returning the version correctly.

Have not tried other actions as do not trust it.

**Comment by ConorWebb96** [comment]
Hey @gmentzik,

The latest version of the CLI is 2.11.34 not 33.
<img width="422" alt="Screenshot 2023-10-16 at 12 31 23" src="https://github.com/Budibase/budibase/assets/126772285/b61e9acf-afc9-4c9f-8867-2fa8d61d407e">

When i return the version using `budi --version` it seems to return correctly. Can you update to 2.11.34 and see if this issue has been resolved.

Thanks.

**Comment by gmentzik** [comment]
Hi @ConorWebb96 ,

The wget command  fetches version from github 2.11.33 for me
Please see example below:
_wget https://github.com/Budibase/budibase/releases/latest/download/cli-linux
--2023-10-16 12:38:58--  https://github.com/Budibase/budibase/releases/latest/download/cli-linux
Resolving github.com (github.com)... 140.82.121.4
Connecting to github.com (github.com)|140.82.121.4|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://github.com/Budibase/budibase/releases/download/2.11.33/cli-linux [following]
--2023-10-16 12:39:04--  https://github.com/Budibase/budibase/releases/download/2.11.33/cli-linux_


Also at releases I don't see 2.11.34:
https://github.com/Budibase/budibase/releases/

Update: 
On a test environment I have installed budi-cli with npm and indeed it prints version 2.11.34. So something is different with linux cli.
